appserver/extras/certificate-management/pom.xml

```
@@ -44,9 +44,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>fish.payara.extras</groupId>
-        <artifactId>extras</artifactId>
-        <version>6.2023.4-SNAPSHOT</version>
+        <groupId>fish.payara.server.core</groupId>
+        <artifactId>core-parent</artifactId>
+        <version>6.0.0-SNAPSHOT</version>
```

Should the relative path not be updated?

---

Wrong version.
Relative path still wrong.

```suggestion
        <version>6.2023.5-SNAPSHOT</version>
        <relativePath>../../../core/core-parent/pom.xml</relativePath>
```

---

Thanks, forgot about cert-management (again)